cribl_stream/README.md

```diff
@@ -0,0 +1,72 @@
+# Cribl Stream
+
+## Overview
+[Cribl Stream][1] helps you process machine data – logs, instrumentation data, application data, metrics, and so on – in real time, and deliver it to your analysis platform of choice. It allows you to:
+
+- Add context to your data, by enriching it with information from external data sources.
+- Help secure your data, by redacting, obfuscating, or encrypting sensitive fields.
+- Optimize your data, per your performance and cost requirements.
+
+Visualize your Cribl Stream metrics in Datadog with the out-of-the-box dashboard.
+
+
+## Data Collected
+
+### Metrics
+
+See [metadata.csv][5] for a list of metrics provided by this integration.
+
+### Events
+
+The Cribl Stream integration does not include any events.
+
+### Service Checks
+
+The Cribl Stream integration does not include any service checks.
+
+
+### Dashboards
+
+Use the out-of-the-box dashboard to view the performance of the Stream with base metrics like events per second, bytes per second, input types, output types, and infrastructure metrics. Monitor reduction percentages by events or bytes, which is useful for improving search performance or licensing and infrastructure costs for the systems of analysis.
+
+## Setup
+You can send your Cribl Stream [internal metrics][6] to the Datadog API, allowing you to monitor your applications and infrastructure without the Datadog Agent. 
+
+### Installation
+
+#### Datadog
+Navigate to _API Keys_ under _Organization Settings_ and create an API Key for Cribl to send data.
+
+#### Cribl
```

Comment:
OAuth instructions will go here - Once you have tested oauth, please add the steps here

Reply:
The self-hosted version of Cribl Stream will not be integrating with the OAuth requirement.